docs/_rpc/ns-clique.md

```diff
@@ -54,15 +54,34 @@ Retrieves the state snapshot at a given block.
 | Console | `clique.getSnapshotAtHash(blockHash)`                    |
 | RPC     | `{"method": "clique_getSnapshotAtHash", "params": [blockHash]}` |
 
+### clique_getSigner
+
+Returns the signer for a specific clique block. Can be called with either a blocknumber, blockhash or an rlp encoded blob.
+The RLP encoded blob can either be a block or a header.
+
+| Client  | Method invocation                                    |
+|:--------|------------------------------------------------------|
+| Console | `clique.getSigner(blockNrOrHashOrRlp)`               |
+| RPC     | `{"method": "clique_getSigner", "params": [string]}` |
+
 ### clique_getSigners
 
-Retrieves the list of authorized signers at the specified block.
+Retrieves the list of authorized signers at the specified block number.
 
 | Client  | Method invocation                                          |
-|:--------|------------------------------------------------------------
+|:--------|------------------------------------------------------------|
 | Console | `clique.getSigners(blockNumber)`                           |
 | RPC     | `{"method": "clique_getSigners", "params": [blockNumber]}` |
 
+### clique_getSignersAtHash
```

Comment:
ERm, what's the point if we have `getSigner` which is a superset that can also accept a hash?

Reply:
I checked and it seems like `getSignersAtHash` is the same as `getSignerS` as in it returns a list of all authorized signers (not the author of a given block as in `getSigner`).